src/bin/iq.rs

```diff
@@ -241,6 +247,67 @@ fn obtain_iq_client(iq_sub_command: &ArgMatches) -> IQClient {
     return IQClient::new(server.clone(), user, token, stage, application, attempts);
 }
 
+fn print_iq_policy_violations(res: PolicyReportResult, parser: &impl ParseToml) -> () {
+    let policy_violations: Vec<Component> = res
+        .components
+        .clone()
+        .into_iter()
+        .filter(|p| {
+            if p.violations
+                .as_ref()
+                .unwrap_or(&(vec![] as Vec<Violation>))
+                .is_empty()
+            {
+                return false;
+            } else {
+                return true;
+            }
+        })
+        .collect();
+
+    if policy_violations.len() > 0 {
+        println!(
+            "Components ({}) with policy violations found",
+            policy_violations.len()
+        );
+        println!("");
+
+        for comp in policy_violations {
+            println!("Package URL: {}", comp.package_url);
+            match comp.violations {
+                Some(violations) => {
+                    println!(
+                        "Known violations: {}",
+                        violations
+                            .into_iter()
+                            .map(
+                                |v| policy_violation_to_styled_object(v.policy_name as String)
+                                    .to_string()
+                            )
```

Comment:
I tried locally and it looks like the `as String` isn't necessary.

```suggestion
                            .map(|v| policy_violation_to_styled_object(v.policy_name).to_string())
```